README.md

```diff
@@ -10,7 +10,7 @@ Found uses [Redux](https://redux.js.org/) for state management and [Farce](https
 
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
```

Comment:
what happened here? i liked the ToC header. it seems like this comment is now duped?